### DISCUSSION TITLE
Do not disable camera?

### DISCUSSION THREAD

**Discussion by mcbeef** [TobiasPankner/Teams-Auto-Joiner#67] [Q&A]
Hello, really like the idea of the script.

Is there a way to make it so the Camera isn't disabled? I want to use this for an unattended device, so its always joined to a meeting and broadcasting video. 

I saw the microphone option in config, but not camera.

**Comment by TobiasPankner** [comment]
The mute_audio option in the config mutes the outgoing audio and not the microphone.  
You can just modify the code at line 407 to say "false" instead of "true". I don't know if this will work with camera permissions though.
